contrib/coreos/user-data.example

```diff
@@ -34,6 +34,22 @@ coreos:
       LimitNOFILE=40000
   - name: fleet.service
     command: start
+  - name: docker-tcp.socket
+    command: start
+    enable: true
+    content: |
+      [Unit]
+      Description=Docker Socket for the API
+
+      [Socket]
+      ListenStream=2375
+      BindIPv6Only=both
+      Service=docker.service
+
+      [Install]
+      WantedBy=sockets.target
+
+
```

Comment:
Let's remove these two blank lines. We don't seem to do that elsewhere in this file.